integration-tests/ci-visibility/run-mocha.mjs

```diff
@@ -4,8 +4,17 @@ import { fileURLToPath } from 'url'
 const mocha = new Mocha({
   parallel: !!process.env.RUN_IN_PARALLEL
 })
-mocha.addFile(fileURLToPath(new URL('./test/ci-visibility-test.js', import.meta.url)))
-mocha.addFile(fileURLToPath(new URL('./test/ci-visibility-test-2.js', import.meta.url)))
+
+if (process.env.TEST_REGEX) {
```

Comment:
nit: `TEST_REGEX` doesn't seem like a good name for passing json data? Is not a regular expression right?

Reply:
yeah it is a regex for `jest` but not for `mocha`. It sucks as a name, but since we run the same tests for both mocha and jest and jest was first, that's what we have. I'll see if I can come up with something better.

Reply:
something like `State` or `Payload` or `Arguments`.. I'm bad at naming, but maybe something generic.

Reply:
changed it to `TESTS_TO_RUN`